plugins/GSdx/GSTextureCache.h

```diff
@@ -104,11 +105,12 @@ class GSTextureCache
 		void Update();
 	};
 
+	using FastSourcePtrList = FastList<Source*, 4>;
```

Comment:
@ssakash what do you mean by exploiting? It's one of the intended usages of C++11 `using` - better replacement for `typedef` with saner semantics (see typedefing function ptrs) and better support for templates. http://en.cppreference.com/w/cpp/language/type_alias